lib/std.dl

```diff
@@ -80,6 +80,7 @@ extern function pow32(base:'A, exp: bit<32>): 'A
  */
 
 #[rust="serde(from=\"Option<A>\", into=\"Option<A>\", bound(serialize=\"A: Clone+Serialize\"))"]
+#[custom_from_record]
```

Comment:
I would add here a comment indicating what this means: the implementation must be given in the corresponding rust file, and it must have two functions with some signatures.